dags/analyzer_helper/discord/discord_extract_raw_infos.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+from dags.analyzer_helper.discord.extract_raw_info_base import ExtractRawInfosBase
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class DiscordExtractRawInfos(ExtractRawInfosBase):
+    def __init__(self, platform_id: str):
+        super().__init__(platform_id)
+        self.client = MongoSingleton.get_instance().client
+        self.db = self.client[self.get_platform_id()]
+        self.collection = self.db['rawmemberactivities']
+
+    def extract(self, period: datetime, recompute: bool = False) -> list:
+        if recompute:
+            return list(self.collection.find({}))
```

Comment:
In `recompute = True` mode we're fetching the data after the `period`
In `recompute = False` mode we would start processing the raw data after the latest processed document on `rawmemberactivities`.

Reply:
For the case of `recompute = True` we can just fetch all data. In this case we're skipping to use `period` and we might not need it in the method input. @cyri113 Also, What's your idea about this?